crypto/engine/build.info

```diff
@@ -5,4 +5,4 @@ SOURCE[../../libcrypto]=\
         tb_rsa.c tb_dsa.c tb_dh.c tb_rand.c \
         tb_cipher.c tb_digest.c tb_pkmeth.c tb_asnmth.c tb_eckey.c \
         eng_openssl.c eng_cnf.c eng_dyn.c \
-        eng_rdrand.c
+        eng_rdrand.c eng_rndr.c
```

Comment:
A new engine isn't going to be accepted into 3.0.
It won't go into 1.1.1 since it's a feature not a bug fix.

Reply:
This is unfortunate... we'll end up having to get it into distros regardless, it's a pretty important "feature"

Reply:
removed engine, fixed